.github/workflows/talk-to-shaka-bot.yaml

```diff
@@ -0,0 +1,100 @@
+name: Talk to Shaka Bot (PRs)
+
+# Runs when comments are created or edited.  Jobs below will be filtered to
+# only run on PRs, not regular issues.
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  handle_command:
+    # Only runs on PRs that contain '@shaka-bot' comments.
+    # Note that contains() is not case sensitive.
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '@shaka-bot')
+    runs-on: [ubuntu-latest]
+    env:
+      # This token must have `repo` scope.
+      GH_TOKEN: ${{ secrets.SHAKA_BOT_TOKEN }}
+      THIS_REPO: ${{ github.repository }}
+      COMMENTER: ${{ github.event.comment.user.login }}
+      PR_NUMBER: ${{ github.event.issue.number }}
+      COMMENT_BODY: ${{ github.event.comment.body }}
+    steps:
+      - name: Handle command
+        shell: bash
+        run: |
```

Comment:
Since it's a pretty big chunk of shell code, would it be better to move it to a separate .sh file?

Reply:
I had convinced myself somehow that this was faster, since it didn't involve cloning the repo... but it's not meaningfully faster.  A shallow clone of the repo in another workflow only took 3s.

Breaking it out would be much more readable, and would enable manual testing.

Reply:
Done.